healthcare.py

```diff
@@ -0,0 +1,1949 @@
+    Unselected colummns (requires individual analyzation ) -> age, avg_glucose_level
+
+'''
+
+# df['gender'] = df['gender'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['gender'].unique()) 
+
+##Conclusion: -> Feature Column -> Content: ['Male' 'Female' 'Other'] -> Requires OHE
+
+# df['age'] = df['age'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['age'].unique()) #-> Feature Column  -> There is a lot of ages must analyze sepreatly  
+##Conclusion: -> Feature Column  -> There is a lot of ages must analyze sepreatly  
+
+# df['hypertension'] = df['hypertension'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['hypertension'].unique()) #->  Feature Column -> Content: [0 1] -> No need for OHE
+##Conclusion: -> Feature Column -> Content: [0 1] -> No need for OHE
+
+# df['heart_disease'] = df['heart_disease'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['heart_disease'].unique()) 
+##Conclusion: -> Feature Column -> Content: [1 0] -> No need for OHE 
+
+# df['ever_married'] = df['ever_married'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['ever_married'].unique()) #->  Feature Column -> Content: ['Yes' 'No'] -> Requires OHE 
+##Conclusion: -> Feature Column -> ['Yes' 'No']. Will require OHE
+
+# df['work_type'] = df['work_type'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['work_type'].unique()) #->  Feature Column -> Content:['Private' 'Self-employed' 'Govt_job' 'children' 'Never_worked'] -> Requires OHE
+##Conclusion: -> Feature Column -> ['Private' 'Self-employed' 'Govt_job' 'children' 'Never_worked']. Will require OHE
+
+# df['Residence_type'] = df['Residence_type'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['Residence_type'].unique())
+##Conclusion: -> Feature Column  -> ['Urban' 'Rural'] -> will require  OHE
+
+# df['avg_glucose_level'] = df['avg_glucose_level'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['avg_glucose_level'].unique()) 
+#Conclusion: -> Feature Column -> [228.69 202.21 105.92 ...  82.99 166.29  85.28]. There is a lot of avg_glucose_levels must analyze sepreatl
+
+# df['bmi'] = df['bmi'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['bmi'].unique())   
+##Conclusion: -> Feature Column -> There is a lot of avg_glucose_levels must analyze sepreatl
+
+# df['smoking_status'] = df['smoking_status'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['smoking_status'].unique()) 
+##Conclusion: -> Feature Column -> ['formerly smoked' 'never smoked' 'smokes' 'Unknown']. Will require OHE
+
+# df['stroke'] = df['stroke'].replace(r'^\s*$', np.nan, regex=True)
+# print(df['stroke'].unique())  
+##Conclusion: -> Label Column -> This is the lablel data [1 0]. It is categorical. Therefore we will apply a classification problem
+
+
+
+'''
+Section 4.3
+-----------
+Analze the rows
+-Drop empty rows if no additonal fillings can be made
+-I believe only column bmi has empty rows

```

Comment:
You could use .describe to actually know what counts of each column are and other info.

Reply:
Makes sense